portfolio-website/script.js

```diff
@@ -1,23 +1,79 @@
+// navBar
 const navBarList = [
-    "Ingredients", "Steps to Make", "Links to Other Wonton Soup Recipes", "About Me"
+    "About Me", "Projects", "Contact"
 ]
 
 for (let i = 0; i < navBarList.length; i++) {
     const label = document.createElement('a');
     label.innerText = navBarList[i];
     label.addEventListener('mouseover', function() {
-        label.style.color = "white";
+        label.style.color = "black";
     })
     label.addEventListener('mouseout', function() {
-        label.style.color = "black";
+        label.style.color = "white";
     })
     label.addEventListener('click', function() {
         label.setAttribute('href', `#${navBarList[i].split(" ").join("")}`);
-        if (i === 3) {
-            label.setAttribute('href', "./aboutme.html");
+        if (i === 2) {
```

Comment:
insted of having i === 2, could we just change the filename to contact? would help w readability